src/modules/Credential/containers/CredentialRecipientModal/useCredentialRecipientModal.tsx

```diff
@@ -0,0 +1,97 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { addCredentialRecipientAdaptor, Schema, Attribute } from 'src/core/adaptors';
+import { CredentialClaims } from 'src/core/api';
+import { emailPattern, urlPattern } from 'src/core/helpers/regexs';
+import * as yup from 'yup';
+
+const recipientSchema = yup
+  .object()
+  .shape({
+    recipient_email: yup
+      .string()
+      .email()
+      .matches(emailPattern, 'This field must be a valid Email')
+      .required('This field is required'),
+    recipient_firstName: yup.string().required('This field is required'),
+    recipient_lastName: yup.string().required('This field is required'),
+  })
+  .required();
+
+const createValidationSchema = (schemaAttributes: Attribute[]) => {
+  const shape: { [key: string]: any } = {};
+  const validations = {
+    TEXT: yup.string().required('This field is required'),
+    NUMBER: yup.number().typeError('This field must be a number').required('This field is required'),
+    BOOLEAN: yup.boolean().default(false),
+    URL: yup.string().matches(urlPattern, 'This field must be a valid URL').required('This field is required'),
+    EMAIL: yup
+      .string()
+      .email()
+      .matches(emailPattern, 'This field must be a valid Email')
+      .required('This field is required'),
+    DATETIME: yup.date().typeError('This field must be a valid date').required('This field is required'),
```

Comment:
Please useTranslation for these, otherwise I have to do another translation task later :)))

Reply:
Yeah it's old version code and we don't have translation for schema too and i forgot to add